src/Http/Livewire/TeamTransferForm.php

```diff
@@ -83,15 +86,25 @@ public function transferTeam(TransferTeam $transferrer)
             ]);
         }
 
+        $teamMember = Jetstream::findUserByEmailOrFail($this->transferTeamForm['email']);
+
+        if ($this->team->hasUser($teamMember)) {
```

Comment:
You need to invert the logic here as you only want to display a warning if the user isn't a member of the team. 

```suggestion
        if (!$this->team->hasUser($teamMember)) {
```